automated-comments/go/hamming/rune_index_used_with_bytes.md

```diff
@@ -0,0 +1,12 @@
+You are iterating over `runes` but you are using the index to get single `bytes` from the string. 
+This is working fine as the exercise only requires you to handle `ascii` characters. 
+Try adding the following test to see what happens if `non-ascii` characters get involved:
+
+```go
+{
+	s1:          "aüa",
+	s2:          "aÃa",
+	want:        1,
+	expectError: false,
+}
+```
```

Comment:
> Iterating over `runes` and using the index to get single `bytes` is safe in this case, because the input is all ASCII characters. To see what happens when non-ASCII characters are used, add this test case: